i18n_ui_amplify.coffee

```diff
@@ -1,7 +1,7 @@
 # Add amplify helper to TAPi18n
 TAPi18n.setLanguageAmplify = (language) ->
   amplify.store 'tap-i18n-language', language
-  TAPi18n.setLanguage language
+  TAPi18n.setLanguage(language).always(TAPi18n.afterUILanguageChange)
```

Comment:
Why `always` and not `done`? If setLanguage will fail the language won't change.

Also, I'd suggest storing the language in `amplify.store 'tap-i18n-language'` only when language changed successfully.